src/test/java/org/codehaus/plexus/components/io/filemappers/FileMapperTest.java

```diff
@@ -164,6 +164,22 @@ public void testPrefixMapper() throws Exception
         mapper.setPrefix( prefix );
         testFileMapper( mapper, SAMPLES, results );
     }
+    
+    public void testSuffixMapper() throws Exception
+    {
+        final String suffix = "suffix";
+        String[] results = getIdentityResults();
+        testFileMapper( new SuffixFileMapper(), SAMPLES, results );
+        testFileMapper( (SuffixFileMapper) lookup( FileMapper.ROLE, SuffixFileMapper.ROLE_HINT ), SAMPLES, results );
+       
+        results = new String[] {null, null, "asuffix", "xyzsuffix.gif", "b/asuffix", "b/xyzsuffix.gif", "b\\asuffix", "b\\xyzsuffix.gif", "csuffix.c/a", "csuffix.c/xyz.gif", "csuffix.c\\a", "csuffix.c\\xyz.gif"};
```

Comment:
Splinting on two lines would make it more readable and  there is missing space after the curly brace.